pkg/telemetry/perf_unix.go

```diff
@@ -44,10 +44,11 @@ func (p *PerfProfile) GetMemoryUsage() map[string]string {
 	props := map[string]string{
 		allocatedmem: strconv.FormatUint(bToMb(m.Alloc), 10),
 		sysmem:       strconv.FormatUint(bToMb(m.Sys), 10),
-		othersysmem:  strconv.FormatUint(bToMb(m.OtherSys), 10),
 		heapallocmem: strconv.FormatUint(bToMb(m.HeapAlloc), 10),
 		heapobjects:  strconv.FormatUint(m.HeapObjects, 10),
```

Comment:
Can you include `HeapIdle` and `HeapInuse`? I think they will be more useful signals.